network/src/main/java/uk/gov/android/network/api/ApiRequest.kt

```diff
@@ -2,21 +2,24 @@ package uk.gov.android.network.api
 
 import uk.gov.android.network.client.ContentType
 
-sealed class ApiRequest {
+sealed class ApiRequest(
+    var apiHeaders: List<Pair<String, String>> = emptyList()
+) {
     data class Get(
         val url: String,
         val headers: List<Pair<String, String>> = emptyList()
-    ) : ApiRequest()
+    ) : ApiRequest(headers)
 
     data class Post<T>(
         val url: String,
         val body: T?,
         val headers: List<Pair<String, String>> = emptyList(),
```

Comment:
Do we need headers re-declared here?

Reply:
As discussed ApiRequest now reverted to original structure but with headers field in FormUrlEncoded data class